### PR TITLE
Fix the problems with prepopulate

### PR DESCRIPTION
**Pull request by erfanio**
Filters out all cancelled tickets.
and can handle not having answer to some questions

### CODE REVIEW

File: app/services/eventbrite.js [hunk 1]
```diff
@@ -33,25 +33,22 @@ const findCreateUniFromName = (name, t) => {
...
+  const { answers } = attendee;
+  const getAnswer = key => _.get(_.find(answers, ['question_id', key]), 'answer', null);
 
-  const qId = env.EVENTBRITE_QUESTION_IDS;
-  const uniName = questions[qId.uni].answer;
-  const studyLevel = questions[qId.study_level].answer;
-  const shirtSize = questions[qId.shirt_size].answer;
-  const dietaryReq = questions[qId.dietary_req].answer;
-  const medicalReq = questions[qId.medical_req].answer;
+  const uniName = getAnswer(questions.uni);
```
Comment (terencehuynh) [inline app/services/eventbrite.js]: So I'm assuming the client passes in the key id for this question.

Just a heads up: medical and dietary requirements were removed from Eventbrite in favour of the Monash off-campus event form - unless you want to hook up Google Spreadsheets too.

File: config/app.yml [hunk 2]
```diff
@@ -8,13 +8,6 @@ default:
   postmark_client_key: "#{process.env.POSTMARK_CLIENT_KEY}"
   jwt_key: this-is-secret
 
-  eventbrite_questions:
```
Comment (terencehuynh) [inline config/app.yml]: I dunno, maybe put this in some _secret_ repo where we can keep track of this?

File: app/services/eventbrite.js (hunk 1 above)
Comment (erfanio) [reply app/services/eventbrite.js]: If we've permanently moved this data I can removed all of this from the app.

File: config/app.yml (hunk 2 above)
Comment (erfanio) [reply config/app.yml]: The reason this is removed is mostly to adapt to different events without restarting/swapping config. Leaking question IDs isn't a big problem afaik

**Pull request merged**